mandatory/3-retrying-random-numbers.js

```diff
@@ -11,8 +11,17 @@ function generateRandomNumber() {
 
 function getRandomNumberGreaterThan50() {
     // TODO - implement using a do-while loop
+    let num = generateRandomNumber();
+    
+    while (num < 50){
+        console.log(num)
+        num = generateRandomNumber()   
+    }
+    return num
+    
 }
```

Comment:
Comment: I understand and can follow the logic on this ones, but not part of my homework. Well done!